Meadowlark-js/backends/meadowlark-postgresql-backend/src/repository/QueryHelper.ts

```diff
@@ -88,12 +153,49 @@ export function getDocumentInsertOrUpdateSql(
   return documentSql;
 }
 
+// SQL for Deletes
+
+/**
+ * Returns the SQL query for deleting a document from the database
+ * @param documentId the document to delete from the documents table
+ * @returns SQL query string to delete the document
+ */
+export function deleteDocumentByIdSql(documentId: string): string {
```

Comment:
Most of your SQL builder functions have a "get" prefix, but this one doesn't. I actually like how this reads better, as the "get" prefixes feel redundant. It's already clear from the "Sql" suffixes that they return SQL.

Reply:
I've gone back and forth on the naming of these functions, they didn't originally have the get or sql has part of the name, and thus looked like they would do the work and not just return the SQL. then I added the 'get' to resolve and when I wasn't satisfied, I added the sql suffix. I'm still not super happy but see your point with not having the 'get' so I'll remove them